### ISSUE TITLE
[Adjust Rule] SIM908 doesn't work for python 3.7

### ISSUE THREAD

**Issue by kasium** [MartinThoma/flake8-simplify#159]
## Desired change

* **Rule(s)**: SIM908 
* **Adjustment**: should work reliable across different python versions

## Explanation
For python3.7 the rule doesn't work when f-strings are somehow used in functions. However, in pyrhon3.11 it works as expected

## Example
```python
import os

def get_framework_info():
    if f"foo" in os.environ:
        port = os.environ[f"foo"]
```

python3.7: _nothing_
python3.11: `Use 'os.environ.get(f"""foo""")' instead of 'if f"""foo""" in os.environ: os.environ[f"""foo"""]'`
